front/src/components/student-registration/student-registration-page/index.js

```diff
@@ -0,0 +1,3 @@
+import StudentRegistrationPage from './student-registration-page';
```

Comment:
Why do we need `student-registration-page.js` at all? Please, move all the code from `student-registration-page.js` to the `index.js` inside `student-registration` folder

Reply:
Removed to index.js